config/root/clusterworkspacetype-organization.yaml

```diff
@@ -1,7 +1,11 @@
-apiVersion: tenancy.kcp.dev/v1alpha1
-kind: ClusterWorkspaceType
-metadata:
-  name: organization
-spec:
-  initializers:
-  - tenancy.kcp.dev/organization
+  apiVersion: tenancy.kcp.dev/v1alpha1
+  kind: ClusterWorkspaceType
+  metadata:
+    name: organization
+  spec:
+    initializers:
+    - tenancy.kcp.dev/organization
+    defaultSubWorkspaceType: Team
+    allowedSubWorkspaceTypes:
+    - Team
+    - Universal
```

Comment:
(no action required) (global) missing newline at EOF (can configure editor to do this automatically)